apps/backlite/boot.js

```diff
@@ -11,26 +11,36 @@
   //Set LCD to zero every reboot
   let s = require("Storage").readJSON("setting.json", 1) || {};
   s.brightness = 0;
+  if (!("lcdTimeout" in s)) s.lcdTimeout = 5; // fallback so  logic doesn't break
   require("Storage").writeJSON("setting.json", s);
   //remove large settings object from memory
-  delete s;
+  s=null;
```

Comment:
We're probably better keeping `delete s;` here, then the variable is gone entirely instead of one existing and holding onto `null`